### DISCUSSION TITLE
Welcome to react-insta-stories Discussions!

### DISCUSSION THREAD

**Discussion by mohitk05** [mohitk05/react-insta-stories#173] [Announcements]
## 👋 Welcome!
  We'll use Discussions as a place to connect with members of our community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  **To get started, you can comment below with an introduction of yourself and share how you are using this component.**

Over the past year, this component has been used by a lot of people and for surprisingly unique applications. I'd love to know more about how you are using this, and also look for potential co-owners of the repository. I can use help in maintaining this project. Let's also prioritize the features you'd like and bring any critical bugs to notice.

I'll also use this space to announce new additions and fixes so that any changes are clearly communicated. Nice to meet you all and thank you for the support until now! 🎉

**Comment by brunolcarlos** [comment]
Is there some react native version for this ?

Thank you

**Reply by mohitk05** [reply]
Not as of now, but I'm experimenting here #180

**Comment by famrullah11** [comment]
I want to ask, how can I add a sound icon at the bottom so that it can mute the sound if the type is a video?
